book/src/validator-doppelganger.md

```diff
@@ -43,9 +43,9 @@ DP works by staying silent on the network for 2-3 epochs before starting to sign
 Staying silent and refusing to sign messages will cause the following:
 
 - 2-3 missed attestations, incurring penalties and missed rewards.
-- 2-3 epochs of missed sync committee contributions (if the validator is in a sync committee, which is unlikely), incurring penalties and missed rewards.
 - Potentially missed rewards by missing a block proposal (if the validator is an elected block
     proposer, which is unlikely).
+- Sync committee contributions are not delayed as those are not slashable
```

Comment:
```suggestion

Notably, sync committee contributions are not slashable and will continue to be produced even when DP is suppressing other messages.
```

The heading of the previous list is "...refusing to sign messages will cause..." so I figured we'd break it out into it's own little paragraph.

Reply:
Sounds good to me, and likely a bit clearer for users to understand